diff --git a/jz.studio.client/libraries/apps/jz-technical-analysis/src/lib/charts/base-chart/base-chart.component.ts b/jz.studio.client/libraries/apps/jz-technical-analysis/src/lib/charts/base-chart/base-chart.component.ts
index 9220617..c508050 100644
--- a/jz.studio.client/libraries/apps/jz-technical-analysis/src/lib/charts/base-chart/base-chart.component.ts
+++ b/jz.studio.client/libraries/apps/jz-technical-analysis/src/lib/charts/base-chart/base-chart.component.ts
@@ -57,7 +57,7 @@ export abstract class BaseChartComponent implements OnChanges, AfterViewInit {
   get panelBadgeWidth(): number {
     if (this.crosshairService.state().readout) {
       switch (this.chartType) {
-        case ChartType.OHLC: return 302;
+        case ChartType.OHLC: return 430;
         case ChartType.VOLUME: return 126;
         case ChartType.MACD: return 196;
         case ChartType.RSI: return 92;
@@ -78,11 +78,15 @@ export abstract class BaseChartComponent implements OnChanges, AfterViewInit {
       switch (this.chartType) {
         case ChartType.OHLC:
           return [
-            { label: this.formatDate(readout.date), className: 'legend-title' },
+            { label: 'PRICE', className: 'legend-title' },
+            { label: `  ${this.formatDate(readout.date)}`, className: 'legend-value' },
             { label: `  O ${this.formatValue(readout.open)}`, className: 'legend-value' },
             { label: `  H ${this.formatValue(readout.high)}`, className: 'legend-value' },
             { label: `  L ${this.formatValue(readout.low)}`, className: 'legend-value' },
-            { label: `  C ${this.formatValue(readout.close)}`, className: 'legend-value' }
+            { label: `  C ${this.formatValue(readout.close)}`, className: 'legend-value' },
+            { label: '  SMA 20', className: 'legend-sma-20' },
+            { label: '  SMA 50', className: 'legend-sma-50' },
+            { label: '  SMA 150', className: 'legend-sma-150' }
           ];
         case ChartType.VOLUME:
           return [
