diff --git a/jz.studio.client/libraries/apps/jz-technical-analysis/src/lib/technical-analysis-panel-preferences.ts b/jz.studio.client/libraries/apps/jz-technical-analysis/src/lib/technical-analysis-panel-preferences.ts
--- a/jz.studio.client/libraries/apps/jz-technical-analysis/src/lib/technical-analysis-panel-preferences.ts
+++ b/jz.studio.client/libraries/apps/jz-technical-analysis/src/lib/technical-analysis-panel-preferences.ts
@@ -34,7 +34,7 @@ export const DEFAULT_PANEL_PREFERENCES: PanelPreference[] = [
     chartType: ChartType.MACD,
     order: 3,
     visible: true,
-    ratio: 0.15,
+    ratio: 0.13,
     showAxisLeft: true,
     showAxisRight: false,
     showXAxisTop: false,
@@ -46,7 +46,7 @@ export const DEFAULT_PANEL_PREFERENCES: PanelPreference[] = [
     chartType: ChartType.RSI,
     order: 4,
     visible: true,
-    ratio: 0.15,
+    ratio: 0.17,
     showAxisLeft: true,
     showAxisRight: false,
     showXAxisTop: false,
